clang/lib/Frontend/ASTMerge.cpp

```diff
@@ -48,7 +48,7 @@ void ASTMergeAction::ExecuteAction() {
                                     /*ShouldOwnClient=*/true));
     std::unique_ptr<ASTUnit> Unit = ASTUnit::LoadFromASTFile(
         ASTFiles[I], CI.getPCHContainerReader(), ASTUnit::LoadEverything, Diags,
-        CI.getFileSystemOpts(), CI.getHeaderSearchOptsPtr());
+        CI.getFileSystemOpts(), CI.getHeaderSearchOpts());
```

Comment:
@adrian-prantl I think this change would mean that `LoadFromASTFile` would no longer modify the header search options on `CI` here.  I'm not familiar with the ASTMerge code; do you know if that is okay?

Reply:
Ping, and CC @zygoloid too.